app/api/search/specs/index.spec.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-lines */
```

Comment:
Please, remove this line, file no longer exceeds max-line limit.

Please, don't use this exception, we have talked about it, and we still want to know this file could be improved by refactoring, or other means to reduce max lines.